assets/content/cookbook/Beginner/reflection-method-call.md

```diff
@@ -0,0 +1,36 @@
+[tags]: / "reflection,dead-code-elimination"
+
+# Invoke object method by string
+
+To invoke method by it's name you will need to use Reflection API.
+
+Snippet bellow shows how to use a string as a object method identifier to invoke it.
+
+## Usage
+```haxe
+class MyClass {
+    public function new () {}
+    @:keep public function printName() {
+        trace("MyClass printName is invoked");
+    }
+}
+
+class Main {
+    static function main() {
+        var myObject = new MyClass();
+        var f = Reflect.field(myObject, "yo");
```

Comment:
What is the purpose of using "yo" here?

Reply:
Thanks. It was working name. I fixed it.